onnxruntime/contrib_ops/cuda/moe/ft_moe/moe_kernel.cu

```diff
@@ -548,7 +573,7 @@ size_t CutlassMoeFCRunner<T, WeightType, Enable>::getWorkspaceSize(int num_rows,
   total_ws_bytes += buf_size * sizeof(T);                    // permuted_data
   total_ws_bytes += padded_experts * sizeof(int64_t);        // Hold total_rows_before_expert_
   total_ws_bytes += num_softmax_outs * sizeof(T);
-  const int bytes_for_fc1_result = interbuf_size * sizeof(T);
+  const int bytes_for_fc1_result = has_fc3_ ? 2 * interbuf_size * sizeof(T) : interbuf_size * sizeof(T);
```

Comment:
why f1 output need double when f3 is available? 
If it is caused by top 2 experts, we shall have an attribute for that.

Reply:
it is used to store the activation result from the optional fc3 weights/bias

Reply:
I see.